hosts.example

```diff
@@ -0,0 +1,30 @@
+# This is an example of minimal tendrl-ansible host inventory file, aligned
+# with instructions in README.md file. For description of installation steps
+# and full list of ansible variables, see the README.md file and readme files
+# of tendrl-ansible roles.
+# This example includes 4 *Tendrl Storage Nodes* (GlusterFS servers) and one
```

Comment:
I'm not sure, how we are referring the Storage nodes on other places, but *"Gluster Storage Servers" (Monitored by Tendrl)* would sound slightly more clear to me. In the meaning, that the storage servers are primarily ordinary "Gluster Storage servers" and the fact that they are/will be monitored by Tendrl is only additional information.

Reply:
Good point.

Reply:
That said, in the README, we have:

>  `tendrl-ansible.tendrl-storage-node`: installation of **Tendrl Storage Node**